src/main/scala/nl/knaw/dans/easy/stage/fileitem/FileItemSettings.scala

```diff
@@ -80,7 +76,31 @@ object FileItemSettings {
       pathInDataset = Some(pathInDataset),
       format = format,
       title = title,
-      subordinate = "objectSDO" -> "dataset"
+      subordinate = "objectSDO" -> "dataset",
+      accessibleTo = accessibleTo,
+      visibleTo = visibleTo
+    )
+
+  /** new folder for a new dataset */
+  def apply(sdoSetDir: File,
+            ownerId: String,
+            pathInDataset: File
+           ) =
+    // no need to catch exceptions thrown by the constructor as the defaults take care of valid values
+    new FileItemSettings(
+      sdoSetDir = Some(sdoSetDir),
+      file = None,
```

Comment:
An attempt resulted in compiler error: `overloaded method value apply with alternatives:`